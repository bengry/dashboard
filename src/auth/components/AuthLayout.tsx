import React from 'react';
import Paper from '@material-ui/core/Paper';
import { makeStyles } from '@material-ui/core/styles';

const useStyles = makeStyles((theme) => ({
  wrapper: {
    paddingTop: theme.spacing(8),
    width: 'auto',
    display: 'block', // Fix IE 11 issue.
    [theme.breakpoints.up(400 + theme.spacing(6))]: {
      width: 400,
      marginLeft: 'auto',
      marginRight: 'auto',
    },
  },
  paper: {
    marginTop: theme.spacing(8),
    display: 'flex',
    flexDirection: 'column',
    alignItems: 'center',
    padding: `${theme.spacing(2)}px ${theme.spacing(3)}px ${theme.spacing(
      3
    )}px`,
  },
}));

const AuthLayout: React.FC = ({ children }) => {
  const classes = useStyles();
  return (
    <div className={classes.wrapper}>
      <Paper className={classes.paper}>{children}</Paper>
    </div>
  );
};

export default AuthLayout;
