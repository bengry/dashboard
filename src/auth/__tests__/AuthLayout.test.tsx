import React from 'react';
import AuthLayout from '../components/AuthLayout';
import { shallow } from 'enzyme';

describe('<AuthLayout />', () => {
  it('should render correctly', () => {
    expect(shallow(<AuthLayout />)).toMatchSnapshot();
  });
});
