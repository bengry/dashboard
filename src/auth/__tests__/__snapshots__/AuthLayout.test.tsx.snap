// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AuthLayout /> should render correctly 1`] = `
<div
  className="makeStyles-wrapper-1"
>
  <WithStyles(ForwardRef(Paper))
    className="makeStyles-paper-2"
  />
</div>
`;
